CMakeLists.txt

```diff
@@ -0,0 +1,94 @@
+project(ATF)
+cmake_minimum_required(VERSION 3.11)
```

Comment:
Need to update base README with new CMake instructions. Also, I encounter this issue when attempting to run this:
```
CMake Error at CMakeLists.txt:29 (find_package):
  By not providing "FindQt5.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Qt5", but
  CMake did not find one.

  Could not find a package configuration file provided by "Qt5" with any of
  the following names:

    Qt5Config.cmake
    qt5-config.cmake

  Add the installation prefix of "Qt5" to CMAKE_PREFIX_PATH or set "Qt5_DIR"
  to a directory containing one of the above files.  If "Qt5" provides a
  separate development package or SDK, be sure it has been installed.


-- Configuring incomplete, errors occurred!
See also "/home/jacobkeeler/sdl_atf/CMakeFiles/CMakeOutput.log".
```

Reply:
You will also need to update the README with the updated config structure.

Reply:
@jacobkeeler README is updated with new Cmake instructions and updated configuration structure. (see 9733c7c0e4a3a495016cabe5e49ab123c3439c02)